codepipeline.yml

```
@@ -0,0 +1,209 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Codepipeline for Frontend repository
+
+Parameters:
+  AlisAppId:
+    Type: String
+  GitHubOwner:
+    Type: String
+    Description: GitHub Owner Name
+    Default: 'AlisProject'
+  GitHubToken:
+    Type: String
+    Description: GitHub Token
+  Repository:
+    Type: String
+    Description: GitHub Repository
+    Default: 'frontend-application'
+  Branch:
+    Type: String
+    Description: GitHub Branch
+    Default: 'master'
+  ServiceName:
+    Description: Name for the service
+    Type: String
+    Default: "alis-frontend-application"
+    AllowedPattern: "[-_ a-zA-Z0-9]*"
+    ConstraintDescription: can contain only alphanumeric characters, spaces, dashes
+      and underscores.
+  CodeBuildImage:
+    Type: String
+    Default: "aws/codebuild/standard:1.0"
+    Description: Image used for CodeBuild project.
+  ApprovalEmailAddress:
+    Type: String
+  region:
```

こちらですが、先頭は大文字であるべきかと思いました。
```suggestion
  Region:
```